### DISCUSSION TITLE
Buscando grupo para Hackaton!

### DISCUSSION THREAD

**Discussion by Carlosdm1** [Launch-X-Latam/MisionFrontEnd#526] [Team Posts]
Hey! Hola a todos, soy Carlos, de Colombia y estoy buscando grupo para Hackaton :D.

**Comment by josa6f** [comment]
podemos formar un grupo, veo que estan en busqueda de un equipo como yo, mi nombre es josafath, soy de la ciudad de México @Carlosdm1 @Gantz9 @Cristoper-ds  ya esperar a que alguien mas se quiera unir, como ven?

**Comment by Cristoper-ds** [comment]
Me parece bien!!

**Comment by josa6f** [comment]
mi correo es lx02172@innovaccion.mx

**Comment by AleMunozF** [comment]
Hola, les falta 1? me puedo unir? Saludos desde Chile! mi correo lx12880@innovaccion.mx

**Comment by josa6f** [comment]
ok, perfecto @AleMunozF, ya solo falta que confirmen los demás

**Comment by Carlosdm1** [comment]
De una! les parece bien si nos conectamos en un chat en teams o prefieren en otra red? @AleMunozF @josa6f @Cristoper-ds @Gantz9

**Comment by josa6f** [comment]
me parece perfecto

**Comment by Salcedotoro** [comment]
Estoy sin grupo también. Me pueden agregar: lx09513@innovaccion.mx

**Comment by FFPeralta** [comment]
Hola Soy Fernando de Buenos Aires. me gustaria formar un grupo con ustedes. lx10575@innovaccion.mx

**Comment by josa6f** [comment]
Saben de cuantos integrantes son los grupos?

**Comment by Gabriand** [comment]
El sherpa dijo de 5
